zeppelin-server/src/main/java/org/apache/zeppelin/socket/NotebookServer.java

```diff
@@ -152,6 +151,9 @@ public void onMessage(NotebookSocket conn, String msg) {
           case ANGULAR_OBJECT_UPDATED:
             angularObjectUpdated(conn, notebook, messagereceived);
             break;
+          case IMPORT_NOTEBOOK:
```

Comment:
Can we have uniform naming? All the other operations are called `XYZ_NOTE`. Let's follow that here as well.